improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -180,7 +172,9 @@ def dry_adiabatic_pressure(
     Returns:
         Array of final pressures (Pa)
     """
-    return initial_pressure * (final_temperature / initial_temperature) ** (1 / KAPPA)
+    return initial_pressure * (final_temperature / initial_temperature) ** (
+        1 / (consts.R_DRY_AIR / consts.CP_DRY_AIR)
```

Comment:
Why not just write this as `consts.CP_DRY_AIR / consts.R_DRY_AIR`?

Reply:
No reason. So I've simplified the equation as suggested.